x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_summary_view.tsx

```diff
@@ -7,171 +7,335 @@
-        title: field,
-        type,
-      },
-      description: {
-        eventId,
-        index,
-        provider,
-        timelineId,
-        value,
-        data: fieldsData,
-        browserField,
-      },
+      fieldsData,
+      type,
+      provider,
+      index,
+      field,
+      browserField,
+      value,
     };
   });
+
+  const [investigation, indicator] = partition(parsedEnrichments, ({ type }) =>
+    isInvestigationTimeEnrichment(type)
+  );
+
+  return (
+    <>
+      {indicator.length > 0 && (
+        <>
+          <EuiPanel hasBorder paddingSize="s" grow={false}>
+            <ThreatSummaryPanelHeader
+              title={i18n.INDICATOR_ENRICHMENT_TITLE}
+              toolTipContent={i18n.INDICATOR_TOOLTIP_CONTENT}
+            />
+
+            {indicator.map(({ fieldsData, index, field, provider, browserField, value }) => (
+              <EnrichedDataRow
+                key={field}
+                field={field}
+                value={
+                  <EnrichmentDescription
+                    eventId={eventId}
+                    index={index}
+                    provider={provider}
+                    timelineId={timelineId}
+                    value={value}
+                    data={fieldsData}
+                    browserField={browserField}
+                    isDraggable={isDraggable}
+                  />
+                }
+              />
+            ))}
+          </EuiPanel>
+        </>
+      )}
+
+      {indicator.length > 0 && investigation.length > 0 && <EuiSpacer size="m" />}
+
+      {investigation.length > 0 && (
+        <>
+          <EuiPanel hasBorder paddingSize="s" grow={false}>
+            <ThreatSummaryPanelHeader
+              title={i18n.INVESTIGATION_ENRICHMENT_TITLE}
+              toolTipContent={i18n.INVESTIGATION_TOOLTIP_CONTENT}
+            />
+
+            {investigation.map(({ fieldsData, index, field, provider, browserField, value }) => (
+              <EnrichedDataRow
+                key={field}
+                field={field}
+                value={
+                  <EnrichmentDescription
+                    eventId={eventId}
+                    index={index}
+                    provider={provider}
+                    timelineId={timelineId}
+                    value={value}
+                    data={fieldsData}
+                    browserField={browserField}
+                    isDraggable={isDraggable}
+                  />
+                }
+              />
+            ))}
+          </EuiPanel>
+        </>
+      )}
+    </>
+  );
 };
 
-const columns: Array<EuiBasicTableColumn<ThreatSummaryItem>> = [
-  {
-    field: 'title',
-    truncateText: false,
-    render: EnrichmentTitle,
-    width: '220px',
-    name: '',
-  },
-  {
-    className: 'flyoutOverviewDescription',
-    field: 'description',
-    truncateText: false,
-    render: EnrichmentDescription,
-    name: '',
-  },
-];
+const HostRiskDataBlock: React.FC<{
```

Comment:
we could consider splitting this component (as well as the proposed `ThreatEnrichment` component above) into separate files and unit test them, as this file is growing a bit